modules/loader/src/main/java/com/opengamma/strata/loader/csv/LoadedCurveSettings.java

```diff
@@ -99,7 +99,8 @@ static LoadedCurveSettings of(
       CurveExtrapolator extrapolatorLeft,
       CurveExtrapolator extrapolatorRight) {
 
-    return new LoadedCurveSettings(curveName, xValueType, yValueType, dayCount, interpolator, extrapolatorLeft, extrapolatorRight);
+    return new LoadedCurveSettings(
```

Comment:
Shouldn't this be:

    return new LoadedCurveSettings(
        curveName, 
        xValueType, 
        yValueType, 
        dayCount, 
        interpolator, 
        extrapolatorLeft, 
        extrapolatorRight);